image.go

```diff
@@ -1185,6 +1340,70 @@ func (rc *RegClient) imageExportDescriptor(ctx context.Context, r ref.Ref, desc
 	return nil
 }
 
+// ImageImportS pushes an image from a tar file (ImageExportS) to a registry.
+func (rc *RegClient) ImageImportS(ctx context.Context, rs []ref.Ref, fileName string, tarFileIO io.ReadSeeker, opts ...ImageOpts) error {
```

Comment:
This should also avoid copying all of the other ImageImport method contents, and instead one method should call the other.

Reply:
Have merged！